5.37.0/upgradeLegacyRteToCmsFeatureFlag.ts

```diff
@@ -0,0 +1,56 @@
+import { Files } from "../utils";
+import { Project } from "ts-morph";
+import { Context } from "../types";
+import { featureFlagExist } from "../utils/featureFlagExist";
+import { addFeatureFlag } from "../utils/addFeatureFlag";
+
+interface Params {
+    files: Files;
+    project: Project;
+    context: Context;
+}
+
+export const upgradeLegacyRteToCmsFeatureFlag = async (params: Params) => {
+    const { project, files, context } = params;
+
+    context.log.info(
+        "Start migrating feature flag for legacy Rich Text editor support in the Headless CMS app."
+    );
+
+    const webinyProjectFile = files.byName("./webiny.project.ts");
+    const source = project.getSourceFile(webinyProjectFile.path);
+
+    const FEATURE_FLAG_NAME = "allowCmsLegacyRteInput";
+
+    if (!source) {
```

Comment:
BTW this check is fine, but even if you didn't include it, it would be fine. If this file was missing, then that would mean something seriously went wrong with user's project.